make/data/asan/asan_default_options.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Google and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, 2025, Google and/or its affiliates. All rights reserved.
```

Comment:
@MBaesken unless you work for Google, or someone from Google requests that you update their copyright notice (as we do for Oracle copyright) then you should not modify other people's copyright notices.

Reply:
Maybe someone from Google can comment what they want/prefer for the headers?
(or was there a statement that they do not want changes to the year info)